tools/busco/VERSION

```diff
@@ -1 +1 @@
-1.2.0
+1.3.0
```

Comment:
This is a minor change, can we version to 1.2.1?

Reply:
OK done